source/Calamari/Deployment/PackageRetention/Repositories/JsonJournalRepository.cs

```diff
@@ -68,8 +71,21 @@ void Load()
             if (File.Exists(journalPath))
             {
                 var json = File.ReadAllText(journalPath);
-                journalEntries = JsonConvert.DeserializeObject<List<JournalEntry>>(json)
-                                            .ToDictionary(entry => entry.Package, entry => entry);
+
+                if (TryParseJournal(json, out var journalContents))
+                {
+                    journalEntries = journalContents.ToDictionary(entry => entry.Package, entry => entry);
+                }
+                else
+                {
+                    var journalFileName = journalPath.Substring(0, journalPath.LastIndexOf(".", StringComparison.Ordinal));
+                    var backupJournalPath = $"{journalFileName}_{DateTimeOffset.UtcNow:yyyyMMddTHHmmss}.json"; // eg. PackageRetentionJournal_20210101T120000.json
+
+                    log.Warn($"The existing package retention journal file {journalPath} could not be read. The file will be renamed to {backupJournalPath}. A new journal will be created.");
+                    File.Move(journalPath, backupJournalPath);
```

Comment:
There's an overload for File.Move that accepts a boolean to overwrite the file if it already exists, which would be good to use - otherwise it will throw an exception.  The odds of this happening aren't high, but they're not zero either.

Reply:
Doesn't exist in 4.0 😢 https://docs.microsoft.com/en-us/dotnet/api/system.io.file.move?view=netframework-4.0

I'll use the alternative they suggest seeing as we should have a small file

> In all .NET versions, you can call Copy(String, String, Boolean) to copy with overwrite, then call Delete to remove the excess source file. This strategy is advisable if the file being copied is small, and you are looking for an "atomic" file operation. If you Delete the file first, and the system or program crashes, the destination file will no longer exist.

Reply:
Ah, Net40 is such a pain!